### DISCUSSION TITLE
How to poll from battery operated devices?

### DISCUSSION THREAD

**Discussion by maddhin** [volker76/tuya_mqtt.net#16] [Q&A]
Hi,

is there a way to poll or simply receive data from battery operated devices (only direction from device to tuya)?

**Comment by volker76** [comment]
As there is no port open to communicate with those low power devices, the only way is retrieving the data from the cloud.
[see Wiki with cloud settings](https://github.com/volker76/tuya_mqtt.net/wiki/GUI-operation-for-TUYA-devices#cloud-mode)
